css-brochure-form/styles.css

```diff
@@ -0,0 +1,70 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  width: 65%;
+  margin: auto;
+}
```

Comment:
You don't ever want to set a `width` on the `body` element, it should always be allowed to take the full viewport width. I see you already set a width on the `form`, that should do the job on its own, this can likely just be deleted.